RELEASING.md

```diff
@@ -1,33 +1,44 @@
 - [Overview](#overview)
 - [Branching](#branching)
-  - [OpenSearch Branching](#opensearch-branching)
+  - [OpenSearch and OpenSearch Dashboards Branching](#opensearch-and-opensearch-dashboards-branching)
   - [Plugin Branching](#plugin-branching)
+  - [Single Repo Artifacts Branching](#single-repo-artifacts-branching)
   - [Feature Branches](#feature-branches)
+  - [Backporting](#backporting)
 - [Versioning](#versioning)
   - [Version Numbers](#version-numbers)
+    - [OpenSearch and OpenSearch Dashboards Version Numbers](#opensearch-and-opensearch-dashboards-version-numbers)
```

Comment:
in our 2024 roadmap, we plan to decouple opensearch release and opensearch dashboards release. would suggest to check that and think through if we should define process to couple these two

Reply:
This is as part current architecture. @seraphjiang Once we have proposal for decoupling accepted and executed for the project - we can revisit then.